CHANGELOG.md

```diff
@@ -9,7 +9,15 @@
 
 ### Breaking changes
 
-* Lorem ipsum.
+* Added support for compound sort and distinct queries.
+    - Multiple consecutive calls to sort or distinct compound on each other
+      in the order applied rather than replacing the previous one.
+    - The order that sort and distinct are applied can change the query result.
+    - It is now an error to define an ascending order on a distinct descriptor, if the
+      order is important then a sort should be applied before the distinct.
+    - It is now an error to call sort or distinct with an empty descriptor, this
+      could previously be used to clear a sort or distinct operation.
```

Comment:
I will remove that restriction and make it a no-op. Just thought it would help people migrate their code.
@kneth do we need a way to remove or clear previously applied sort/distinct operations?